editor/src/components/canvas/canvas-strategies/strategies/flow-reorder-helpers.ts

```diff
@@ -27,43 +28,70 @@ export function isValidFlowReorderTarget(
   }
 }
 
-export function areAllSiblingsInOneDimension(
+export function areAllSiblingsInOneDimensionFlexOrFlow(
   target: ElementPath,
   metadata: ElementInstanceMetadataMap,
+  targetLayout: 'flex' | 'flow',
```

Comment:
I wonder if this should just be based on the metadata for `target`, since you are using the metadata anyways to get subsequent information, such as flexDirection